src/test/java/Homework19.java

```diff
@@ -0,0 +1,16 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    public void deleteCreatedPlaylist() throws InterruptedException {
+        login();
+        clickSubmit();
+        selectEmptyPlaylist();
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223042708-80786695-472f-4417-a564-7f2bc0dbb6e8.png)

Reply:
can you please try again now.
thanks